### PR TITLE
VW MQB: Škoda Octavia Mk3

### PR DESCRIPTION
**Pull request by jyoung8607**
**Škoda Octavia Mk3**

Covers all variants of the third generation Octavia (2013-2019).

- [x] added to README
- [x] test route added to [test_routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/test/test_models.py)
- [x] route with openpilot: 66e5edc3a16459c5|2021-05-25--19-00-29

Thanks to community Octavia owners cben (2015) and some friend of Saber422's (2019).

### CODE REVIEW

File: selfdrive/test/test_routes.py [hunk 1]
```diff
@@ -442,6 +442,10 @@
     'carFingerprint': VOLKSWAGEN.SKODA_KODIAQ_MK1,
     'enableCamera': True,
   },
+  "66e5edc3a16459c5|2021-05-23--23-44-46": {
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:445]: need to look into why CI passed, but this route isn't uploaded

Comment (adeebshihadeh) [reply selfdrive/test/test_routes.py:445]: make sure the user has the upload raw toggle on

Comment (jyoung8607) [reply selfdrive/test/test_routes.py:445]: I noticed that. Not only that, but it passed on an earlier commit where I forgot to merge the route in entirely, which should have failed immediately.

**Comment by jyoung8607** [comment]
Kicking back to draft till I can confirm we have a viable route.

**Comment by jyoung8607** [comment]
I withdrew the test route. At the time of the drive, the device was running a fork that doesn't disengage on brake. Hopefully someone else with an Octavia and modern hardware will make themselves known. There are a bunch of Octavia folks, unfortunately they mostly have older hardware and/or are located in Germany.

@adeebshihadeh: CI should be failing because Octavia isn't in `non_tested_cars`, and it's still passing. I haven't tried to track down why just yet. Might look later tonight or tomorrow if you aren't already aware of the cause.

**Comment by jyoung8607** [comment]
Ready for review per our DMs.

**Pull request merged**